### PR TITLE
add postcss & plugins

### LINKED ISSUE #8: use postcss by default
with: 
- rucksack
- lost-grid

open to other plugins as well - http://postcss.parts/

### PR DESCRIPTION
**Pull request by hhsnopek**
closes #8

### CODE REVIEW

File: init.coffee [hunk 1]
```diff
@@ -13,11 +13,30 @@ exports.configure = [
     message: 'Do you need netlify hosting?'
   },
   {
+    type: 'list'
+    name: 'css'
+    message: 'What would you like to use as your CSS Preprocessor?'
+    choices: ['postcss', 'stylus']
```
Comment (kylemac) [inline init.coffee]: my vote is we use stylus and postcss by default, actually. since that's how we practically use it now

Comment (hhsnopek) [inline init.coffee]: Ohh, together?

Comment (hhsnopek) [inline init.coffee]: using [seaneking/poststylus](https://github.com/seaneking/poststylus)?

Comment (kylemac) [inline init.coffee]: yeah, i know it's weird right now and you can use plugins for each to do the same things, but think of it as a slow transition to postcss - but we still depend on a lot from stylus, like axis.

Comment (kylemac) [inline init.coffee]: i haven't seen poststylus before, but i was just thinking that we have both the `stylus.use` and the `postcss.use` and trust the developer to use them sensibly

Comment (hhsnopek) [inline init.coffee]: I'm actually using poststylus in share-button: https://github.com/carrot/share-button/blob/master/gulpfile.js#L24

Comment (kylemac) [inline init.coffee]: well shit, let's go with that. wouldn't be tough to add postcss/stylus separately if we ever encounter a problem

File: root/package.json [hunk 2]
```diff
@@ -26,9 +26,27 @@
...
+    <% if (grid == true) { %>
+      "jeet": "6.1.x",
+      "rupture": "0.6.x",
+      "stylus": "0.52.x",
+      "lost": "6.6.x",
+    <%} -%>
+    <% for(var i = 0; i < postcssPLugins.length; i++) { %>
+      <% if (postcssPlugins[i].checked) { %>
+        <% if (postcssPLugins[i].name === "rucksack") { %>
+          , "rucksack-css": "0.8.x"
```
Comment (hhsnopek) [inline root/package.json]: @kylemac the prepended `,` doesn't work for json - any ideas on a better way to structure this?

Comment (kylemac) [inline root/package.json]: Write a temp json file, then require it, then remove it with the after hook?

Comment (hhsnopek) [inline root/package.json]: solid plan

**Comment by hhsnopek** [comment]
@kylemac this just needs a review then I can rebase/squash & merge

**Comment by kylemac** [comment]
:+1:

**Pull request merged**